sdks/java/extensions/protobuf/src/main/java/org/apache/beam/sdk/extensions/protobuf/ProtoSchemaTranslator.java

```diff
@@ -157,6 +157,7 @@ static Schema getSchema(Class<? extends Message> clazz) {
 
   static Schema getSchema(Descriptors.Descriptor descriptor) {
     Set<Integer> oneOfFields = Sets.newHashSet();
```

Comment:
Since the variable names oneOfFields and oneOfFieldLocationMap are very similar, we should disambiguate them somehow. Maybe emphasize that this one refers to the components of a oneof field, not the oneof field itself.

We should also add comments explaining the meaning of these variables.
```suggestion
    Set<Integer> oneOfComponentFields = Sets.newHashSet();
```